manifests/b/Benjamin Vollmer/kafui/0.1.23/github.com.benny93.kafui.installer.yaml

```
@@ -0,0 +1,34 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+PackageIdentifier: github.com.benny93.kafui
```

```suggestion
PackageIdentifier: benny93.kafui
```
Please update file path (between `/b/` and `/kafui/`) to match this change.